app/models/travel_request.rb

```diff
@@ -3,7 +3,33 @@
 #  - attributes are defined on Request
 #  - behavior can be defined here
 class TravelRequest < Request
+  include AasmConfig
+
   def to_s
     "#{creator}  #{event_requests.first.recurring_event.name}"
   end
+
+  aasm column: "status" do
+    # add in an additional state changes_requested which is only valid for a travel request
+    state :changes_requested
+    event :change_request do
+      transitions from: :changes_requested, to: :changes_requested
```

Comment:
Just curious, is this to add additional requested changes by supervisor(s)?

Reply:
Good catch!  I missed removing that.  At first I was thinking about allowing all states to transition to themselves, but then realized that was only needed for pending.  I forgot to remove it here.